test/worker_external_task_test.py

```diff
@@ -127,15 +128,6 @@ def test_external_dependency_completes_later(self):
                                                            'retry-external-tasks',
                                                            False) is True
 
-        original_get_work = self.scheduler.get_work
```

Comment:
indeed,
but my patch removes that DIY monkey patching completely, so there is no need for either